lib/delayed_job/delayed_job_callbacks.rb

```diff
@@ -8,21 +8,20 @@ class DelayedJobCallbacks < ::Delayed::Plugin
           block.call(job)
         ensure
           duration = Time.now - start
-          Metrician.gauge("queue.process", duration) if Metrician.configuration[:queue][:process][:enabled]
-          Metrician.gauge("#{job_metric_instrumentation_name(worker)}.process", duration) if Metrician.configuration[:queue][:job_specific][:enabled]
+          Metrician.gauge("jobs.run", duration) if Metrician.configuration[:jobs][:run][:enabled]
+          Metrician.gauge("jobs.run.#{job_metric_instrumentation_name(job)}", duration) if Metrician.configuration[:jobs][:job_specific][:enabled]
```

Comment:
The per job metric name doesn't match my expectations based on https://github.com/Instrumental/metrician/pull/11#discussion_r122048071 . I think I like the way in the message better because it's not currently clear when adding a metric to a graph that `jobs.run` exists when `jobs.run.stuff` exists. The autocomplete implies only the latter.

I'm very open to discussion, though. I think changing how autocomplete works is a good idea, but who knows when we'll get to that.

I also hadn't thought about the error metric names, so maybe the full list looks like this now:
```
jobs.run
jobs.error
jobs.queue.<queue_name>
jobs.queue.error.<queue_name>
jobs.task.<job_name>
jobs.task.error.<job_name>
jobs.queue.<queue_name>.task.<job_class>
jobs.queue.error.<queue_name>.task.error.<job_class>
```

That last one is really weird. I think what I would prefer is:
```
jobs.run
jobs.error
jobs.run.queue.<queue_name>
jobs.error.queue.<queue_name>
jobs.run.task.<job_name>
jobs.error.task.<job_name>
jobs.run.queue.<queue_name>.task.<job_class>
jobs.error.queue<queue_name>.task.<job_class>
```
but I definitely think we'd want to change the autocomplete to make it clearer `jobs.run` exists when `jobs.run.queue.*` exists as well.

What do you think?